pages/index.tsx

```diff
@@ -48,57 +59,57 @@ export default function Home({
 
       <Header socials={socials} />
 
-      <section id='hero' className='snap-start'>
-        <Hero pageInfo={pageInfo}/>
+      <section id="hero" className="snap-start">
+        <Hero pageInfo={pageInfo} />
       </section>
 
-      <section id='portfolio' className='snap-start'>
+      <section id="portfolio" className="snap-start">
         <Portfolio videos={videos} genres={genres} />
       </section>
 
-      <section id='audio' className='snap-start'>
+      <section id="audio" className="snap-start">
         <AudioComponent audio={audio} />
       </section>
 
-      <section id='skills' className='snap-start'>
+      <section id="skills" className="snap-start">
         <Skills skills={skills} />
       </section>
 
-      <section id='experience' className='snap-center'>
+      <section id="experience" className="snap-center">
         <WorkExperience experiences={experiences} />
       </section>
 
-      <section id='about' className='snap-center'>
-        <About pageInfo={pageInfo}/>
+      <section id="about" className="snap-center">
+        <About pageInfo={pageInfo} />
       </section>
-      
-      <section id='contact' className='snap-start'>
-        <Contact pageInfo={pageInfo}/>
+
+      <section id="contact" className="snap-start">
+        <Contact pageInfo={pageInfo} />
       </section>
 
-      <Link href='#hero'>
-        <footer className='sticky bottom-5 w-full cursor-pointer'>
-          <div className='flex items-center justify-center'>
-            <img 
-              className='h-10 w-10 rounded-full filter grayscale hover:grayscale-0 cursor-pointer'
+      <Link href="#hero">
+        <footer className="sticky bottom-5 w-full cursor-pointer">
+          <div className="flex items-center justify-center">
+            <img
+              className="h-10 w-10 rounded-full filter grayscale hover:grayscale-0 cursor-pointer"
               src={urlFor(pageInfo[0]?.profilePic).url()}
-              alt='Photo of Michael and link back to Home page'
+              alt="Photo of Michael and link back to Home page"
             />
           </div>
         </footer>
       </Link>
     </div>
-  )
+  );
 }
 
 export const getStaticProps: GetStaticProps<Props> = async () => {
-  const pageInfo: PageInfo[] = await fetchPageInfo();
-  const experiences: Experience[] = await fetchExperiences();
-  const skills: SkillType[] = await fetchSkills();
-  const socials: Social[] = await fetchSocials();
-  const videos: Video[] = await fetchVideos();
-  const genres: Genre[] = await fetchGenres();
-  const audio: Audio[] = await fetchAudio();
+  const pageInfo = await getPageStaticProps();
+  const experiences = await getExperiencesStaticPops();
+  const skills = await getSkillsStaticProps();
+  const socials = await getSocialsStaticProps();
+  const videos = await getVideoStaticProps();
+  const genres = await getGenreStaticProps();
+  const audio = await getAudioStaticProps();
```

Comment:
use these func only inside getStaticProps or pages/api folder, if you are fetching data in a component use your normal fetch